stereo_points/stereo_points.py

```diff
@@ -219,8 +271,22 @@ def onControlPointSelectedChanged(self, newNode):
             newCoordTable.AddNodeReferenceID("stereotaxia_trajLine", newNode.GetID())
             newNode.AddNodeReferenceID("stereotaxia_coordTable", newCoordTable.GetID())
             # add an observer for both nodes, whenever one is renamed, the other one is renamed as well
-            self.lObs_tableNode = newCoordTable.AddObserver(vtk.vtkCommand.ModifiedEvent, self.onCoordTableModified)
-            newNode.AddObserver(vtk.vtkCommand.ModifiedEvent, self.onControlPointNodeModified)
+            self.observers_list.append(
```

Comment:
When you save the scene and then reload, node references are preserved, so `coordTable_id` will be valid. However, you will not have any observers. Therefore, even if `coordTable_id` is valid, you need to check if you already have observers and if you don't then add these observers.